easybuild/easyconfigs/r/rmats-turbo/rmats-turbo-4.1.1-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,74 @@
+# by Christian Meesters, Johannes Gutenberg-University Mainz
+
+easyblock = "CmdCp"
+
+name = "rmats-turbo"
+version = "4.1.1"
+
+homepage = "https://github.com/Xinglab/rmats-turbo"
+description = """
+            rMATS turbo is the C/Cython version of rMATS
+            (refer to http://rnaseq-mats.sourceforge.net). The major
+            difference between rMATS turbo and rMATS is speed and space
+            usage. rMATS turbo is 100 times faster and the output file is
+            1000 times smaller than rMATS. These advantages make analysis
+            and storage of a large scale dataset easy and convenient.
+              """
+
+toolchain = {"name": "GCCcore", "version": "10.2.0"}
+
+source_urls = ["https://gitlab.com/Xinglab/archive/refs/tags/"]
+sources = ["v%(version)s.tar.gz"]
+patches = ["rmats-turbo-4.1.1.patch"]
+checksums = [
+    "cdfde24b68be0b41b4ca43e5414d3c292deb7efd617d7ee2315e2d6e6bcb1c92",
+    "f42aef1730592d821e8d75557c9b4dc2eb5c226edd6b9064fdd93afb9f2e960d",
+]
+
+builddependencies = [("CMake", "3.18.4"), ("binutils", "2.35")]
+
+dependencies = [
+    ("Python", "3.8.6"),
+    ("SciPy-bundle", "2020.11", "", ("foss", "2020b")),
+    ("OpenBLAS", "0.3.12", "", ("GCC", "10.2.0")),
+    ("ScaLAPACK", "2.1.0", "", ("gompi", "2020b")),
+    ("GSL", "2.6", "", ("GCC", "10.2.0")),
+    ("SAMtools", "1.12", "", ("GCC", "10.2.0")),
+    ("STAR", "2.7.8a", "", ("GCC", "10.2.0")),
```

Comment:
This needs to go into foss instead if it depends from things from foss, else things will break (for at least some module naming schemes)